src/beam_stats_msg_statsd_gauge.erl

```diff
@@ -26,24 +26,24 @@ of_msg_graphite(
     , timestamp = _Timestamp
     }
 ) ->
-    PathBin = beam_stats_msg_graphite:path_to_bin(Path),
-    cons(PathBin, Value).
+    PathIOList = beam_stats_msg_graphite:path_to_iolist(Path),
+    cons(PathIOList, Value).
 
--spec cons(binary(), non_neg_integer()) ->
+-spec cons(iolist(), non_neg_integer()) ->
```

Comment:
I'd like name to be precisely spec'd, which at the moment appears to be `[binary() | string()]`, but maybe it is worth it to go even further and change `#beam_stats_msg_graphite.path` to be `[string()]` rather than `[binary()]`?